src/EditorFeatures/Core/Implementation/EditAndContinue/DebuggerContractVersionCheck.cs

```diff
@@ -4,7 +4,7 @@
 
 using System;
 using System.Runtime.CompilerServices;
-using Microsoft.CodeAnalysis.EditAndContinue.Contracts;
+using Microsoft.VisualStudio.Debugger.Contracts.HotReload;
 
 namespace Microsoft.CodeAnalysis.EditAndContinue
```

Comment:
(GitHub won't let me comment on line 13 so pretend this is comment there)

The tracked work item comments this was until we moved to preview 5 which I would have imagined we were by now, or did a later change break stuff again?

Reply:
Just confirming that we are running on Preview 5.

Reply:
We'll see. Maybe we need more sophisticated condition.